src/types.ts

```diff
@@ -0,0 +1,74 @@
+export const LEFT = "Left";
+export const RIGHT = "Right";
+export const UP = "Up";
+export const DOWN = "Down";
+export type HandledEvents = React.MouseEvent | TouchEvent | MouseEvent;
```

Comment:
This is causing Typescript to import React as a global reference, moving this to an import will fix the issue @bsz2 is facing